src/jsMain/resources/texts/OpenUtau ARPAsing to OpenUtau VCCV.txt

```diff
@@ -45,4 +45,6 @@ t=t
 v=v
 w=w
 y=y
-z=z
\ No newline at end of file
+z=z
```

Comment:
Could you remove this kind of lines where no actual replacement happens? I think the process will get faster.